network-protection/network-protection-api/src/main/java/com/duckduckgo/networkprotection/api/NetworkProtectionAccessState.kt

```diff
@@ -19,34 +19,32 @@ package com.duckduckgo.networkprotection.api
 import com.duckduckgo.navigation.api.GlobalActivityStarter.ActivityParams
 import kotlinx.coroutines.flow.Flow
 
-interface NetworkProtectionWaitlist {
+interface NetworkProtectionAccessState {
 
     /**
-     * Returns the state of the NetP waitlist
+     * Returns the state of the NetP
      * The caller DOES NOT need to specify the dispatcher when calling this method
      */
-    suspend fun getState(): NetPWaitlistState
+    suspend fun getState(): NetPAccessState
 
     /**
      * Returns a flow of the states of the NetP access
      * The caller DOES NOT need to specify the dispatcher when calling this method
      */
-    suspend fun getStateFlow(): Flow<NetPWaitlistState>
+    suspend fun getStateFlow(): Flow<NetPAccessState>
 
     /**
      * Call this method to get the [ActivityParams] corresponding to the activity to launch for the current
-     * state of the waitlist beta
+     * state of the VPN
      * The caller DOES NOT need to specify the dispatcher when calling this method
      *
-     * @return the [ActivityParams] for the activity to launch that corresponds to the current waitlist beta or `null` if no screen is available
+     * @return the [ActivityParams] for the activity to launch that corresponds to the current VPN or `null` if no screen is available
      * for that state
      */
     suspend fun getScreenForCurrentState(): ActivityParams?
 
-    sealed class NetPWaitlistState {
-        object NotUnlocked : NetPWaitlistState()
-        object JoinedWaitlist : NetPWaitlistState()
-        object PendingInviteCode : NetPWaitlistState()
-        data class InBeta(val termsAccepted: Boolean) : NetPWaitlistState()
+    sealed class NetPAccessState {
+        data object NotUnlocked : NetPAccessState()
+        data object InBeta : NetPAccessState()
```

Comment:
NIT: Technically InBeta doesn’t exist anymore. Probably Unlocked is a better naming or something along the lines of that.

Reply:
will do